src/util/models.py

```diff
@@ -522,11 +521,33 @@ def merge_model(cr, source, target, drop_table=True, fields_mapping=None, ignore
     remove_model(cr, source, drop_table=drop_table, ignore_m2m=ignore_m2m)
 
 
-def remove_inherit_from_model(cr, model, inherit, keep=(), skip_inherit=()):
+def remove_inherit_from_model(cr, model, inherit, keep=(), skip_inherit=(), strict=False):
```

Comment:
I know I'm the one who propose this name, but I don't find `strict` clear enough.
What about `only`? `direct_only`?

Reply:
TBH the only keyword that I find "clear enough" is `keep_parent_fields`.
`only` -> only what?
`direct_only` -> direct from where?
`keep_parent_fields` -> it hints that we keep fields, from parents, it really doesn't make sense to think about parents of `model` since what we remove is from `inherit` thus it looks more aligned with what we do.

wdyt?

Reply:
Damn, this is really [hard](https://martinfowler.com/bliki/TwoHardThings.html).

Reply:
That's why I personally don't care much about the actual name as long as the docstrings are clear... the latter is still not sure but we are getting there.